attributes/default.rb

```diff
@@ -15,3 +15,7 @@
 
 # PGP Key Server
 default['le']['pgp_key_server'] = 'pgp.mit.edu'
+
+# Debian Release
+
+default['le']['deb'] = 'trusty'
```

Comment:
Should we not default this to node['lsb']['codename'] and let users override if required?